README.md

```diff
@@ -124,6 +124,13 @@ if (!globalThis.fetch) {
 }
 ```
 
+`node-fetch` is an ESM-only module - you are not able to import it with `require`. If you are unable to use ESM in your project you can use the async `import()` from CommonJS to load `node-fetch` asynchronously:
+
+```js
+// mod.cjs
+const fetch = (...args) => import('node-fetch').then(({default: fetch}) => fetch(...args));
+```
```

Comment:
I would recommend not to use the ESM version of node-fetch at all as long as your code base is CJS, I would recommend instead stick to the 2.x release. I'm afraid this might cause hard to understand/debug bugs. When people create isssues we would need to ask how they import node-fetch besides asking what version they are using.

Reply:
Are you saying we should add a note like: "if you are stuck with cjs, then you can also use a older version of node-fetch `@2.x` or `@v3.0.0-beta.9` that is built with cjs"?

Reply:
Yes. I would only recommend `2.x` though. It's well tested and widely used for a long time now, and we can still ship 2.x releases in case of bugs or security concerns.

Reply:
Adding this after would that be fine?

```diff
+ The alternative is to stick with the `@2.x` branch that is built with cjs 
+ `npm install node-fetch@2`
```

Reply:
I won't die on my hill 😁 

My concern is to reduce maintenance overhead, and implying that you can use `node-fetch@3` with an CJS app using the hack you showed above makes me worry about weird edge cases.



> Adding this after would that be fine?
> 
> ```diff
> + The alternative is to stick with the `@2.x` branch that is built with cjs 
> + `npm install node-fetch@2`
> ```

Adding this is better than not, I'd still vote to not recommend using v3 with CJS apps at all.